cypress/integration/add-sequence/back-to-all-jobs.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Users should be able to navigate back to the job list
```

Comment:
It's not a _super_ expensive test in my opinion. It's easy to maintain and runs in about two seconds. But we could remove it. In general we could look at which paths we want to test as e2e tests and which we don't. Thinking a bit about that would probably be good, because this test suite has just been growing organically for a while.

Reply:
All right, sounds good to me 👍

Reply:
I'll resolve this one as well, since we have a follow up ticket here as well.